openlibrary/templates/covers/book_cover.html

```diff
@@ -19,7 +19,7 @@
             </div>
             <div class="SRPCoverBlank" style="display: $cond(not cover_url, 'block', 'none');">
                 <div class="innerBorder">
-                    <div class="BookTitle">$:macros.TruncateString(title, 70)
+                    <div class="BookTitle">$:macros.TruncateString(title, 30)
```

Comment:
```suggestion
                    <div class="BookTitle">$:macros.TruncateString(title, 45)
```

This is too short; eg https://testing.openlibrary.org/books/OL22595027M/Harry_Potter_and_the_Philosopher's_Stone . 45 seems ok.